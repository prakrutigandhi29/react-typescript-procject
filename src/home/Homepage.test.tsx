import HomePage from '../components/HomePage';
import renderer from 'react-test-renderer';
import { render,screen } from '@testing-library/react';
  describe('<HomePage/>', () => {
    test('renders learn react link', () => {
        render(<HomePage />);
        expect(screen.getByRole('heading')).toHaveTextContent('Home');
      });
    test('snapshot', () => {
        const tree = renderer.create(<HomePage />).toJSON();
        expect(tree).toMatchSnapshot();
      });
     });

