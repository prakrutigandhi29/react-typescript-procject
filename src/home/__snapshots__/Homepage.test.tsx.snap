// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage/> snapshot 1`] = `
<div>
  <h2>
    Home Page
  </h2>
</div>
`;
